subsys/pm/Kconfig

```diff
@@ -117,4 +117,11 @@ config PM_DEVICE_RUNTIME_EXCLUSIVE
 
 endif # PM_DEVICE
 
+
+config PM_SHELL
+	bool "Power Management Shell"
+	depends on SHELL && (PM || PM_DEVICE)
```

Comment:
You need `PM_DEVICE` (actually `PM_DEVICE_RUNTIME` when using device runtime pm). This or is wrong, because if you have PM and !PM_DEVICE you can't do any device power management.

Reply:
Sorry for the confusion I was thinking beyond just device power management to support general power management features like stats. `PM` and `PM_DEVICE` seem to be two independent groups of power management features.